plugins/vfake.py

```diff
@@ -1,11 +1,15 @@
 # to anonymize a column in vd: do "setcol-fake" with e.g. 'name' 'isbn10' or any of the functions on Faker()
 
+import json
+
 from visidata import vd, Column, Sheet, option, options, asyncthread, Progress
 
-__version__ = '0.9'
+__version__ = '0.10'
```

Comment:
When do we hit 1.0?

Reply:
Now?

Reply:
That makes sense to me.  But then this would be `1.0` :)

Reply:
Ha, your comment motivated a nudge from 0.10 to 1.0! I do believe that makes you a 10x developer :).

🥁🤕